presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcRecordCursor.java

```diff
@@ -71,7 +49,36 @@
 
     public JdbcRecordCursor(JdbcClient jdbcClient, JdbcSplit split, List<JdbcColumnHandle> columnHandles)
     {
-        this.columnHandles = ImmutableList.copyOf(requireNonNull(columnHandles, "columnHandles is null"));
+        this.columnHandles = requireNonNull(columnHandles, "columnHandles is null").stream()
```

Comment:
Let's write this as
```java
this.columnHandles = columnHandles.toArray(new JdbcColumnHandle[0]);
```
To avoid the IntelliJ warning `Declaration redundancy | Simplify stream API call chains`

It's also a bit shorter (code wise). Technically, it is doing an extra empty array allocation, but streams seem to do way more allocations anyway. (not that it matters either way here, but for the argument that the stream version does one less allocation)